### ISSUE TITLE
currentSelectedItem property gives undefined !

### ISSUE THREAD

**Issue by cappittall** [a1626/image-carousel#7]
First of all, thank you very much for this useful component.
currentSelectedItem property is not reflectin upon image changes. 

when I go through the codes. I think below code: 
```
          /**
           * Gives index of image on display (count starts from 0).
           */
           currentSelectedItem: {
            type: Number,
            value: 0,
            readOnly: true
          },
```
If I am not wrong, should be 

> notify:true

**Comment by a1626** [comment]
You are right notify property needs to be included. I'm on vacation right now I'll fix, test it ASAP.

**Comment by a1626** [comment]
@cappittall thanks for reporting the issue i've fixed it in v2.0.1.
